lib/remap-lang/src/parser.rs

```diff
@@ -818,9 +811,7 @@ mod tests {
                 ],
             ),
             ("only_fields(.foo,)", vec![" 1:18\n", "= expected argument"]),
-            ("only_fields(,)", vec![" 1:13\n", "= expected argument"]),
-            ("only_fields(.foo,)", vec![" 1:18\n", "= expected argument"]),
-            ("only_fields(,)", vec![" 1:13\n", "= expected argument"]),
+            // ("only_fields(,)", vec![" 1:13\n", "= expected argument"]),
```

Comment:
This errors because of the same reason as the others, except that it considers the `()` as a group expression.

So in this case:

* `only_fields` is considered to be a variable
* `()` is a group
* because `,` isn't valid in a group, the error is: `expected assignment, if-statement, argument, query, or block`

The expected error should be `expected argument` (as seen a few lines above).